demos/SignTransactionPin.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Pin Input</title>
+</head>
+<body>
+<link href="../src/nimiq-style.css" rel="stylesheet">
+<script src="http://cdn.nimiq.com/web-offline.js"></script>
+<script src="../src/lib/EncryptionType.js"></script>
+<script src="../src/lib/Key.js"></script>
+
+<div>
+    PIN is 000000
+</div>
+
+<script type="text/javascript">
+    const keyguard = window.open('../src/sign-transaction/');
+
+    keyguard.onload = async () => {
+        await Nimiq.WasmHelper.doImportBrowser();
+
+        keyguard.KeyStore.prototype.getType = async (address) => {
+            return EncryptionType.LOW;
+        };
+
+        keyguard.KeyStore.prototype.get = async (signer, pin) => {
+            // fake delay
+            await new Promise(resolve => setTimeout(resolve, 1000));
+
+            const buf = "0108708e9b94c9598aa0138b47c139482b61466ff1d403a0e0c18c30f9041085839035528909aa4c76dde86390aec3c189120f77a000";
+
+            return Key.loadEncrypted(buf, pin);
+        };
+
+        const txRequest = {
+            sender: 'NQ82 HP54 C9D4 2FAG 69QD 6Q71 LURR 5187 0V3X',
+            recipient: 'NQ82 HP54 C9D4 2FAG 69QD 6Q71 LURR 5187 0V3X',
```

Comment:
Should use a different recipient than sender, as sending to oneself is not allowed.